__tests__/unit/lib/utils/repoGitDiff.test.js

```diff
@@ -12,25 +12,26 @@ describe(`test if repoGitDiff`, () => {
       stdout: '',
     }))
     // eslint-disable-next-line no-undef
-    const work = repoGitDiff({ output: '', repo: '' }, globalMetadata)
+    const work = repoGitDiff({ output: '', repo: '', ignore: '__tests__/unit/lib/utils/.forceignore' }, globalMetadata)
```

Comment:
use a constant at the beginning of the file for the path to the forceignore file plz